pages/phoneVerification.tsx

```diff
@@ -0,0 +1,87 @@
+import Image from "next/image";
+import React, { ChangeEvent, useState } from "react";
+import ErrorModalHandler from "../components/errorModalHandler";
+import { phoneverification } from "../public/locales/fa.json";
```

Comment:
- Change ```phoneverification``` to ```camelCase``` mode, both in this file and in translation file
- Use ```translate``` in your import instead of defining ```translate``` variable in your component code.
```suggestion
import { phoneVerification as translate } from "../public/locales/fa.json";
```